examples/training/classification/imagenet/basic_training.py

```diff
@@ -74,6 +75,11 @@
     False,
     "Whether or not to use FP16 mixed precision for training.",
 )
+flags.DEFINE_boolean(
```

Comment:
wanna also fix the random generator shape mismatch in this PR?

Reply:
Good call -- done